.run/single-binary.run.xml

```diff
@@ -0,0 +1,13 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="single-binary" type="GoApplicationRunConfiguration" factoryName="Go Application">
+    <module name="flyte" />
+    <working_directory value="$PROJECT_DIR$" />
+    <go_parameters value="-tags console" />
+    <parameters value="start --config=flyte-single-binary-local.yaml" />
```

Comment:
How do I use this file

Reply:
It should show up in Goland as a "go build" config. Here's a screenshot:
![Screenshot 2024-04-30 at 4 21 09 PM](https://github.com/flyteorg/flyte/assets/653394/dc6b983c-021a-4851-83a1-7a1b0f55964c)

Reply:
Awesome, thank you!